src/pg/sql/42_observatory_exploration.sql

```diff
@@ -372,16 +372,28 @@ BEGIN
              geom_type::TEXT,
              geom_extra::JSONB,
              geom_tags::JSONB,
-      $1 = ANY(numers) valid_numer,
-      $2 = ANY(denoms) valid_denom,
-      $3 = ANY(timespans) valid_timespan
-      FROM observatory.obs_meta_geom
+             $1 = ANY(numers) valid_numer,
```

Comment:
AFAIK is not possible. https://www.postgresql.org/docs/9.5/static/sql-prepare.html